<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_bearb_pkopf</class>
 <widget class="QDialog" name="Dialog_bearb_pkopf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Dicke</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>LÃ¤nge</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_l"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_d"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_b"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Breite</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_l</tabstop>
  <tabstop>lineEdit_b</tabstop>
  <tabstop>lineEdit_d</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
